ts/modules/full-screen.src.ts

```diff
@@ -225,7 +225,7 @@ class Fullscreen {
             );
 
             if (chart.container.parentNode instanceof Element) {
-                const promise = chart.container.parentNode[
+                const promise = chart.renderTo[
```

Comment:
The above `if` does not make sense at all now :)

Reply:
Yeah, you are right I will fix it, but first I want to know if there are any obstacles to use `renderTo` rather than `container.parentNode`.

Reply:
I'm fine with it. You should also change the container on the constructor (line 106).